third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -332,7 +353,43 @@ EOH
 `, datasetID, cryptoKeyName, tableID)
 }
 
-<% unless version == 'ga' -%>
+func testAccBigQueryTableHivePartitioning(bucketName, datasetID, tableID string) string {
+	return fmt.Sprintf(`
+resource "google_storage_bucket" "test" {
+	name          = "%s"
+	force_destroy = true
+}
+
+resource "google_storage_bucket_object" "test" {
+	name    = "key1=20200330/init.csv"
+	content = ";"
+	bucket  = google_storage_bucket.test.name
+}
+
+resource "google_bigquery_dataset" "test" {
+		dataset_id = "%s"
+	}
```

Comment:
Looks like spacing got a little mixed up in this config. The resource below is also tabbed in too much